src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DefaultLSPProjectionProvider.cs

```diff
@@ -45,7 +45,7 @@ public override async Task<ProjectionResult> GetProjectionAsync(LSPDocumentSnaps
 
             var languageQueryParams = new RazorLanguageQueryParams()
             {
-                Position = new Position(position.Line, position.Character),
+                Position = position,
```

Comment:
Was there a reason we were doing a deep copy of Position?

Reply:
nope